### ISSUE TITLE
InvalidMixinException on 18.2-1.0.2

### ISSUE THREAD

**Issue by jasonmarco** [jpenilla/better-fabric-console#6]
Using the latest release of better-fabric-console **18.2-1.0.2**, I am getting an `InvalidMixinException`. The server has no other mods other than the two below. Are there any other prerequisites required for this mod?

I am using Minecraft version 1.18.2, with Fabric Loader 0.13.3.

- better-fabric-console-mc1.18.2-1.0.2
- fabric-api-0.48.0+1.18.2

Please see:  [latest.log](https://github.com/jpenilla/better-fabric-console/files/8310519/latest.log)
----
```
Caused by: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: @Shadow field LOGGER was not located in the target class net.minecraft.class_3176. Using refmap better-fabric-console-refmap.json

````

**Comment by Vincydroid** [comment]
same envirooment, samer issue
minecraft 1.18.2, fabric loader 0.13.3
- better-fabric-console-mc1.18.2-1.0.2
- fabric-api-0.48.0+1.18.2
and here's [latest.log](https://github.com/jpenilla/better-fabric-console/files/8311139/latest.log)

**Comment by jpenilla** [comment]
Seems to be some sort of issue with loom... My local copy of 1.0.2 was fine but the published version had the LOGGER field not remapped. Recompiling 1.0.2 yielded a jar broken in the same way as the published version, and then cleaning caches led to building working jars again. I've published [1.0.3](https://github.com/jpenilla/better-fabric-console/releases/tag/v1.0.3) which is essentially just a recompile with clean caches and should work again.
